typespec-extension/src/main/java/com/azure/typespec/model/EmitterOptions.java

```diff
@@ -72,6 +72,9 @@ public class EmitterOptions {
     @JsonProperty(value="dev-options")
     private DevOptions devOptions;
 
+    @JsonProperty(value="package-dir")
+    private String packageDir;
```

Comment:
I don't think we need to put the packageDir into Java part.

I'd prefer the emitter.ts do the config of the default situations, and set the flavor there.
E.g.

```
if (!flavor) {
  if (package-dir.startWith("azure)) {
    flavor = "Azure";
  } else {
    // default, can change later
    flavor = "Azure";
  }
}
```

Reply:
thanks, updated